src/config.ts

```diff
@@ -59,6 +59,7 @@ const page = {
 };
 
 const store = {
+	showOnlySeries: process.env.SHOW_ONLY_SERIES ? process.env.SHOW_ONLY_SERIES.split(',') : ['3080'],
```

Comment:
I'd default to `[]` here, same as for brands to defaul to "show all" (or maybe `['3070','3080','3090']` to hide debug cards)?

Reply:
Yeah, let me change to this `['3070','3080','3090']` as the default.  Using `[]` will include the debug cards and probably not a great idea if someone messes up their environment config